src/app/Home.module.css

```diff
@@ -7,7 +7,7 @@
     margin-right: 1em;
 }
   
-h2 {

```

Comment:
[Comment]: Looks like CSS Modules have a requirement that every styling selector is scoped to an id/class to minimize overlaps; we'll have to keep that in mind moving forward.

Reply:
I will keep that in mind!